src/algebra/module/basic.lean

```diff
@@ -241,6 +241,12 @@ instance semiring.to_module [semiring R] : module R R :=
   zero_smul := zero_mul,
   smul_zero := mul_zero }
 
+@[priority 910] -- see Note [lower instance priority]
```

Comment:
Is there any motivation for this very specific value?

Reply:
I've no idea, I just copied it from the identical instance for right-multiplication.